website/content/en/docs/upgrading/upgrade-guide.md

```diff
@@ -36,60 +36,29 @@ WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE T
-* Breaking API (Manual Migration Needed):
-  * Ubuntu is dropped as a first class supported AMI Family
-  * `karpenter.sh/do-not-consolidate` (annotation), `karpenter.sh/do-not-evict` (annotation), and `karpenter.sh/managed-by` (tag) are all removed. `karpenter.sh/managed-by`, which currently stores the cluster name in its value, will be replaced by eks:eks-cluster-name
-  * The taint used to mark nodes for disruption and termination changed from `karpenter.sh/disruption=disrupting:NoSchedule` to `karpenter.sh/disrupted:NoSchedule`. It is not recommended to tolerate this taint, however, if you were tolerating it in your applications, you'll need to adjust your taints to reflect this.
-* Environment Variable Changes:
-  * Environment Variable Changes
-  * LOGGING_CONFIG, ASSUME_ROLE_ARN, ASSUME_ROLE_DURATION Dropped
-  * LEADER_ELECT renamed to DISABLE_LEADER_ELECTION
-  * `FEATURE_GATES.DRIFT=true` was dropped and promoted to Stable, and cannot be disabled.
-      * Users currently opting out of drift, disabling the drift feature flag will no longer be able to do so.
-* Defaults changed:
-  * API: Karpenter will drop support for IMDS access from containers by default on new EC2NodeClasses by updating the default of `httpPutResponseHopLimit` from 2 to 1.
-  * API: ConsolidateAfter is required. Users couldn’t set this before with ConsolidationPolicy: WhenUnderutilized, where this is now required. Users can set it to 0 to have the same behavior as in v1beta1.
-  * API: All `NodeClassRef` fields are now all required, and apiVersion has been renamed to group
-  * API: AMISelectorTerms are required. Setting an Alias cannot be done with any other type of term, and must match the AMI Family that's set or be Custom.
-  * Helm: Deployment spec TopologySpreadConstraint to have required zonal spread over preferred. Users who had one node running their Karpenter deployments need to either:
-    * Have two nodes in different zones to ensure both Karpenter replicas schedule
-    * Scale down their Karpenter replicas from 2 to 1 in the helm chart
-    * Edit and relax the topology spread constraint in their helm chart from DoNotSchedule to ScheduleAnyway
-  * Helm/Binary: `controller.METRICS_PORT` default changed back to 8080
+Refer to the `v1` Migration Guide for the [full changelog]({{<ref "./v1-migration.md#changelog">}}).
 
 ### Upgrading to `0.37.0`+
 
 {{% alert title="Warning" color="warning" %}}
 `0.33.0`+ _only_ supports Karpenter v1beta1 APIs and will not work with existing Provisioner, AWSNodeTemplate or Machine alpha APIs. Do not upgrade to `0.37.0`+ without first [upgrading to `0.32.x`]({{<ref "#upgrading-to-0320" >}}). This version supports both the alpha and beta APIs, allowing you to migrate all of your existing APIs to beta APIs without experiencing downtime.
 {{% /alert %}}
 
+{{% alert title="Note" color="primary" %}}
+Webhooks have been re-enabled by default starting in `0.37.3` to faciliate migration to `v1.0`.
```

Comment:
For what versions were they disabled by default?

Reply:
`v0.33+`. I don't think we need to add anything additional there since it's still called out in the `v0.33` section of the upgrade guide.